server/server.go

```diff
@@ -336,11 +341,12 @@ func (s *Server) handleReq(conn *gokeyless.Conn, ch chan *gokeyless.Header) {
 func (s *Server) Serve(l net.Listener) error {
 	defer l.Close()
```

Comment:
this never gets executed when program normally exits and one (or more) goroutine is blocked on l.Accept(). Will need to figure out why